src/Migrate/Migrate.Autorest/custom/Get-AzMigrateLocalServerReplication.ps1

```diff
@@ -132,9 +132,9 @@ function Get-AzMigrateLocalServerReplication {
     )
     
     process {
-        $helperPath = Join-Path $PSScriptRoot 'Helper' 'AzLocalCommonSettings.ps1'
+        $helperPath = Join-Path $PSScriptRoot 'Helper\AzLocalCommonSettings.ps1'
```

Comment:
As we discussed, please use `[System.IO.Path]::Combine()` for compatibility on PS 5.1, i.e.


```suggestion
        $helperPath = [System.IO.Path]::Combine($PSScriptRoot, 'Helper', 'AzLocalCommonSettings.ps1')
```